src/Neo/Network/P2P/Payloads/Conflicts.cs

```diff
@@ -38,6 +41,23 @@ public override JObject ToJson()
 
         public override bool Verify(DataCache snapshot, Transaction tx)
         {
+            // Check already stored signers
+
+            var conflictingSigners = tx.Signers.Select(s => s.Account);
+
+            foreach (var attr in tx.GetAttributes<Conflicts>())
+            {
+                var conflictRecord = snapshot.TryGet(
+                    new KeyBuilder(NativeContract.Ledger.Id, LedgerContract.Prefix_Transaction).Add(attr.Hash))?
+                    .GetInteroperable<TransactionState>();
```

Comment:
@shargon, are you sure that this is the proper place for this check? Maybe it would be better to create some `CanFitConflictingSigners` method inside native Ledger contract, move this checkong logic to this method and just call `NativeContract.Ledger.CanFitConflictingSigners` here?